### DISCUSSION TITLE
Update to React 19

### DISCUSSION THREAD

**Discussion by rytix** [mercadopago/sdk-react#115] [Ideas]
I’ve conducted initial testing with React 19, and everything seems to work as expected.

Could you perform additional testing to confirm compatibility with React 19 and update the peerDependencies in package.json to include support for React 19 ?

**Comment by icaldana** [comment]
Hi @rytix,

Thank you for conducting the initial testing with React 19! We will investigate further by performing additional testing to confirm compatibility with React 19 and will update the peerDependencies accordingly. We’re adding this task to our backlog and will address it as soon as possible.

Thank you for your feedback and contribution!

**Comment by icaldana** [comment]
Hello @rytix,

Today we released v0.0.21 which has official React 19 support!

Thank you,
